store/store.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'store'
-    spec.version                  = '5.0.0-alpha01'
+    spec.version                  = '5.0.0-alpha1'
```

Comment:
01 ideally so that we can do alpha 10

Reply:
Ah the last release was 5.0.0-alpha1. Next release we can do 5.0.0-alpha02?